server/lib/item-transform.js

```diff
@@ -0,0 +1,37 @@
+const get = require('lodash.get');
+
+const PRICE_MATCH = '[PRICE]';
+
+module.exports.itemTransform = item => {
+	item.formatted = {};
+
+	if (item.secondaryPricing && item.secondaryPricing.length > 0) {
+		const { copyPricePath, copyTemplate } = item.secondaryPricing[0];
+		const secondaryPriceValue = get(item, copyPricePath);
+
+		item.formatted.secondaryPricing = copyTemplate.replace(PRICE_MATCH, secondaryPriceValue);
+		delete item.secondaryPricing;
+	}
+
+	if (item.pricingCopyPricePath && item.pricingCopyTemplate) {
+		const pricingValue = get(item, item.pricingCopyPricePath);
```

Comment:
what is the advantage of using `get` here?

Reply:
`item.pricingCopyPricePath` is a long variable, that needs to cherry pick a deep value from `item.pricing` and `lodash.get` is the safest piece of code for that, any other implementation might be a bit hard to read, whilst the code is executed in the backend and the cost is low